fall23/deepface/face_matching.ipynb

```diff
@@ -0,0 +1,3368 @@
+{
```

Comment:
<p>An explanation about what these columns mean and how the results differ with other base models could improve understandability. A comparison chart would be very helpful!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-atfal-mafkoda-missing-children/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>